Update New-ADUser.md

msbemba opened the pull request:
Removed the add,remove/replace values as they cannot be used on new-aduser parameter

msbemba commented:
Issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2014

opbld31 commented:
Docs Build status updates of commit _[3dcc7f7](https://github.com/msbemba/windows-powershell-docs/commits/3dcc7f71586ee70766ba51ecc43ca22b5666de06)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/windows/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/windows/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C13%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202007130538556112-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=rvDhqH5N7RIUWjXgQX0p2zgaZ8dWMN2D9wBwFaZHxCk%3D&st=2020-07-13T06%3A11%3A12Z&se=2020-08-13T06%3A16%3A12Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

e0i commented:
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

get-itips commented:
Agree that we need to not only remove but also replace with the current accepted syntax

msbemba commented:
> I think we should add some detail like "Use the following syntax: ,,.... If the values contain spaces or otherwise require quotation marks, use the following syntax: "","",...""." and apply the same for -Certificates parameter.

Thank you @dariomws and @get-itips . I have added the syntax as per your suggestion. let me know if it looks good so that I can apply it on the other versions as well.

Thanks
Sri

opbld31 commented:
Docs Build status updates of commit _[23918f6](https://github.com/msbemba/windows-powershell-docs/commits/23918f6611b48971e62c5e4088cb0765461e9216)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/windows/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/windows/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C15%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202007151951434901-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yTySnF1sP0DnvzQRZFVAG%2B%2FSdKU%2BKrb0K%2FmWu7%2B9n%2BA%3D&st=2020-07-15T20%3A12%3A15Z&se=2020-08-15T20%3A17%3A15Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

e0i commented:
Good to go with @tseward and @get-itips 's approval.

dariomws commented:
> > I think we should add some detail like "Use the following syntax: ,,.... If the values contain spaces or otherwise require quotation marks, use the following syntax: "","",...""." and apply the same for -Certificates parameter.
> 
> Thank you @dariomws and @get-itips . I have added the syntax as per your suggestion. let me know if it looks good so that I can apply it on the other versions as well.
> 
> Thanks
> Sri

@msbemba the latest update is wrong, when I copied and pasted the description from the [New-Mailbox](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/New-Mailbox.md) article, there was a problem with the markdown format. Now I think it should be ok:
To enter multiple values, use the following syntax: \<value1\>,\<value2\>,...\<valueX\>. If the values contain spaces or otherwise require quotation marks, use the following syntax: "\<value1\>","\<value2\>",..."\<valueX\>".

msbemba commented:
> > > I think we should add some detail like "Use the following syntax: ,,.... If the values contain spaces or otherwise require quotation marks, use the following syntax: "","",...""." and apply the same for -Certificates parameter.
> > 
> > 
> > Thank you @dariomws and @get-itips . I have added the syntax as per your suggestion. let me know if it looks good so that I can apply it on the other versions as well.
> > Thanks
> > Sri
> 
> @msbemba the latest update is wrong, when I copied and pasted the description from the [New-Mailbox](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/New-Mailbox.md) article, there was a problem with the markdown format. Now I think it should be ok:
> "To enter multiple values, use the following syntax: <value1>,<value2>,...<valueX>. If the values contain spaces or otherwise require quotation marks, use the following syntax: "<value1>","<value2>",..."<valueX>"."

Thank you @dariomws , I have made changes as per your suggestion.

e0i commented:
Good to go with @tseward and @get-itips approval.

opbld33 commented:
Docs Build status updates of commit _[b81a0f0](https://github.com/msbemba/windows-powershell-docs/commits/b81a0f0b516d1b023f55ecdaba85c2264ecc979f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/windows/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/windows/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C20%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202007201209041496-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=m43KZ7gzNajgFRYFFEOrMPyoJcF6IPZCg9FIgp9p7a8%3D&st=2020-07-20T12%3A29%3A19Z&se=2020-08-20T12%3A34%3A19Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld33 commented:
Docs Build status updates of commit _[344fbcf](https://github.com/msbemba/windows-powershell-docs/commits/344fbcf4a33109e7a804560b63afd3149f57f208)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/windows/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/windows/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C20%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202007201210403048-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=lMgTStnA%2BqlVAxZFdIYTRhdqZTTNNCTYC78k6JoJMqE%3D&st=2020-07-20T12%3A56%3A07Z&se=2020-08-20T13%3A01%3A07Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

get-itips commented:
Current description for -Certificates in New-AdUser is wrong, it seems it was copied from AccountExpirationDate, can you please correct it? New-AdComputer has it ok.

get-itips commented:
Also, I tested  the -Certificates parameter an it is expecting **an array** of X509Certificate objects

```
$cert1 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test1.cer"
$cert2 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test2.cer"
$certs = $cert1,$cert2
New-AdUser -Certificates $certs
```


Please **test this** and change the required format accordingly.

Thanks

msbemba commented:
Thank you, Andres. The Type is already set to X509Certificate[] and we have an example (1) which shows how to use the parameter. We have added the syntax to use when there are multiple values for certificates. 

 

I have also changed the Description for certificate parameter in new-aduser. 

 

Thanks,

Sri

 

From: Andres Mariano Gorzelany <notifications@github.com> 
Sent: Monday, July 20, 2020 6:51 PM
To: MicrosoftDocs/windows-powershell-docs <windows-powershell-docs@noreply.github.com>
Cc: Sriraman M S <msbemba@gmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [MicrosoftDocs/windows-powershell-docs] Update New-ADUser.md (#2049)

 

Also, I tested the -Certificates parameter an it is expecting an array of X509Certificate objects

$cert1 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test1.cer"
$cert2 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test2.cer"
$certs = $cert1,$cert2
New-AdUser -Certificates $certs

Please test this and change the required format accordingly.

Thanks

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2049#issuecomment-661035230> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AK63OZHGTXY2INGPZ65VXJTR4RAD3ANCNFSM4OYF7T7Q> .  <https://github.com/notifications/beacon/AK63OZC3DLTDODYJVC7DDRTR4RAD3A5CNFSM4OYF7T72YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOE5TJRXQ.gif>

opbld33 commented:
Docs Build status updates of commit _[80bd7a8](https://github.com/msbemba/windows-powershell-docs/commits/80bd7a8c3007ed15ca4d51fe66e224ae1574aa9c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/windows/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/windows/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C20%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202007201630141861-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3jhySO7UAKfltVCOYH1SbbTA5hwB50iPgHpwvrP%2BCBE%3D&st=2020-07-20T16%3A50%3A36Z&se=2020-08-20T16%3A55%3A36Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

get-itips commented:
> Thank you, Andres. The Type is already set to X509Certificate[] and we have an example (1) which shows how to use the parameter. We have added the syntax to use when there are multiple values for certificates. I have also changed the Description for certificate parameter in new-aduser. Thanks, Sri 

I understand, however, Does -Certificates supports supplying more than on array of X509Certificate on a list array?

like this:

```
$cert1 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test1.cer"
$cert2 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test2.cer"
$certsA = $cert1,$cert2

$cert3 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test3.cer"
$cert4 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test4.cer"
$certsB = $cert1,$cert2

New-AdUser -Certificates $certsA,$certsB 
```

Can you kindly test that?

Thanks

opbld32 commented:
Docs Build status updates of commit _[41acdcf](https://github.com/msbemba/windows-powershell-docs/commits/41acdcffeca02a3249ff168fc5a965dc5f7c6d7e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/windows/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/windows/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C20%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202007201630270695-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7B3C%2FByGieVCuHc1yE9j1lUmizh2f8hMT%2B7kcYEwyAA%3D&st=2020-07-20T17%3A16%3A03Z&se=2020-08-20T17%3A21%3A03Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

msbemba commented:
> > Thank you, Andres. The Type is already set to X509Certificate[] and we have an example (1) which shows how to use the parameter. We have added the syntax to use when there are multiple values for certificates. I have also changed the Description for certificate parameter in new-aduser. Thanks, Sri
> 
> I understand, however, Does -Certificates supports supplying more than on array of X509Certificate on a list array?
> 
> like this:
> 
> ```
> $cert1 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test1.cer"
> $cert2 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test2.cer"
> $certsA = $cert1,$cert2
> 
> $cert3 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test3.cer"
> $cert4 = New-Object System.Security.Cryptography.X509Certificates.X509Certificate "C:\Certs\Test4.cer"
> $certsB = $cert1,$cert2
> 
> New-AdUser -Certificates $certsA,$certsB 
> ```
> 
> Can you kindly test that?
> 
> Thanks

Thank you, @get-itips  I get the below error when using New-AdUser  GlenJohn -Certificates $certsA,$certsB 

![image](https://user-images.githubusercontent.com/45987684/87974421-aef88480-cae7-11ea-84ec-416f1fc15695.png)


If i use the below it works. 

PS C:\Users\Administrator> New-ADUser GlenJohn -Certificate (new-object System.Security.Cryptography.X509Certificates.X5
09Certificate $certsa,$certsb)

get-itips commented:
Ok thanks for that testing @msbemba so this is not true for Certificates, right?

_To enter multiple values, use the following syntax: <value1>,<value2>,...<valueX>. If the values contain spaces or otherwise require quotation marks, use the following syntax: "<value1>","<value2>",..."<valueX>"."_

msbemba commented:
Yes, I'm not able to add multiple values in one cmdlet like  *"","",...""."*

*I'll update the document accordingly for the certificate section.*

On Tue, Jul 21, 2020 at 1:28 AM Andres Mariano Gorzelany <
notifications@github.com> wrote:

> Ok thanks for that testing @msbemba <https://github.com/msbemba> so this
> is not true for Certificates, right?
>
> *To enter multiple values, use the following syntax: ,,.... If the values
> contain spaces or otherwise require quotation marks, use the following
> syntax: "","",...""."*
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2049#issuecomment-661301901>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AK63OZFFSMYA35GHCSUVJQLR4SOW7ANCNFSM4OYF7T7Q>
> .
>


-- 

-----------------------------------------

Sriraman Srinivasan

If Plan "A" Doesnt work there are 25 other letters:) Stay Cool !

opbld33 commented:
Docs Build status updates of commit _[9ff3b4d](https://github.com/msbemba/windows-powershell-docs/commits/9ff3b4d16d1b3351f484e7e2f0b09a05d6c7db3c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/windows/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/windows/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C22%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202007222251251292-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=62Ri5D%2FOxtvvg9t82Q8JCydiWcQrvXJrpfImfli5S%2BU%3D&st=2020-07-22T23%3A11%3A14Z&se=2020-08-22T23%3A16%3A14Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld33 commented:
Docs Build status updates of commit _[a0f77ed](https://github.com/msbemba/windows-powershell-docs/commits/a0f77ed004a024c3f4bcc3d7c2906817febdce02)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/windows/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/windows/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/windows/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012-ps/activedirectory/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012-ps/activedirectory/New-ADUser.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADComputer.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADComputer.md) | :white_check_mark:Succeeded | |
[docset/winserver2012r2-ps/addsadministration/New-ADUser.md](https://github.com/msbemba/windows-powershell-docs/blob/patch-119/docset/winserver2012r2-ps/addsadministration/New-ADUser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C22%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202007222253387523-2049%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=8624YZbj4vJ4PbBF38QAYvWcvpbBVf%2FGHgrmXwWcpLs%3D&st=2020-07-22T23%3A35%3A47Z&se=2020-08-22T23%3A40%3A47Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
